pkg/awx/connection.go

```diff
@@ -322,18 +324,19 @@ func (c *Connection) rawPost(path string, query url.Values, input []byte) (outpu
 	c.setCredentials(request)
 	c.setContentType(request)
 	c.setAccept(request)
+	if glog.V(2) {
+		glog.Infof("Sending POST request to '%s'.", address)
```

Comment:
If we are dumping the complete request then this ^ line is redundant, as it will be already part of the result of `DumpRequest`.